buildinfo/mode_dev.go

```diff
@@ -0,0 +1,5 @@
+//go:build !production
```

Comment:
I'm a bit unsure about this way of selecting the proxy url.

If understand correctly, the goal of having `test` and `prod` proxies, is:
- Have separate resources between developers and users
- Introduce `staging` endpoint later.

If this is right, then I have some concerns/suggestions.

1. Making 2 endpoints just for separation of resources doesn't sound as right solution

    If credentials are the same on test/prod hosts, then nothing stops users of using `test` proxy.
    If credentials are different, then can't we just split resources on a single host, basing on credentials provided? And if really separate hosts are needed, just introduce some load balancer (again, based on credentials).

2. Selecting the proxy in runtime sounds more convenient

    I assume the user story is to test `staging` before deploying it to `prod`, then QA should be able to test latest release against the `staging` proxy without rebuilding the app.
    
    In this case, I believe a runtime CLI option would work better: `nim_status_client --proxy-rpc-url=staging|prod`. So that would mean to pass the url from client at login time.

Reply:
I do agree that runtime selection probably makes more sense, though seems like build-time selection is easier at this stage.

As for hosts, there's no point in trying to optimize a cost of few euro by spending hours of engineer time worth WAY more than that on re-organizing things to save buying one less host. It makes no sense. As you say, we can use test stage or now for non-release builds, and later ad staging if need be, the cost for one DO host is like 12 bucks.

I do intend to have separate credentials for test and prod, just right now they are the same since I just didn't get around to it.